src/pages/search/advanced_search/ScientificValueFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Checkbox, FormControlLabel } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam } from '../../../api/searchApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+
+enum ScientificValueLevels {
+  LevelZero = 'Unassigned,LevelZero',
+  LevelOne = 'LevelOne',
+  LevelTwo = 'LevelTwo',
+}
+
+export const ScientificValueFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const scientificValueParam = searchParams.get(ResultParam.ScientificValue) ?? '';
+
+  const selectedScientificValues = {
+    levelZero: scientificValueParam.includes(ScientificValueLevels.LevelZero),
+    levelOne: scientificValueParam.includes(ScientificValueLevels.LevelOne),
+    levelTwo: scientificValueParam.includes(ScientificValueLevels.LevelTwo),
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const name = event.target.name as keyof typeof selectedScientificValues;
```

Comment:
Syns dette ser fint ut jeg! eneste som er litt synd er at `<Checkbox name="levelZero"` ikke har noen type i sin name, så der kan vi sette hva som helst feil uten å oppdage det.. ser ikke noen umiddelbar løsning for forenkler mer enn det kompliserer da :/

Reply:
Måtte sjekke litt, beste jeg finner er noe sånt
```
enum LevelField {
  LevelZero = 'levelZero',
  LeveOne = 'levelOne',
  LevelTwo = 'levelTwo',
}

const selectedScientificValues = {
  [LevelField.LevelZero]: scientificValueParam.includes(ScientificValueLevels.LevelZero)
  (...)
}

const name = event.target.name as LevelField;

<FormControlLabel control={<Checkbox name={LevelField.LevelZero} checked={selectedScientificValues.levelZero} />}
        label={t('search.advanced_search.scientific_value.level_zero')}
      />
```

Vet ikke om det er noe særlig bedre, så op ptil deg!